src/frontend/src/components/NavBar/components/AppNavBar/index.tsx

```diff
@@ -36,35 +34,13 @@ const AppNavBar = (): JSX.Element => {
   const group = getCurrentGroupFromUserInfo(userInfo);
   const route = userInfo ? ROUTES.DATA : ROUTES.HOMEPAGE;
 
-  const flag = useTreatments([FEATURE_FLAGS.user_onboarding_v0]);
-  const isUserOnboardingFlagOn = isFlagOn(
-    flag,
-    FEATURE_FLAGS.user_onboarding_v0
-  );
-
   const toggleDropdown: MouseEventHandler = (e) => {
     const newAnchor = isGroupDetailsDropdownOpen ? null : e.currentTarget;
     setAnchorEl(newAnchor);
     setIsGroupDetailsDropdownOpen(!isGroupDetailsDropdownOpen);
   };
 
   const name = group?.name;
-  const orgElements = (
-    <Fragment>
-      <Separator />
-      <NavOrg>{name}</NavOrg>
-    </Fragment>
-  );
-
-  function hasOrg(): JSX.Element | null {
-    if (group === undefined) {
```

Comment:
if it matters here, it is (or, at least it will be in the future) possible for a user to not be a member of any groups.